### ISSUE TITLE
[BUG] Wine application window is not reachable

### ISSUE THREAD

**Issue by fablevi** [pop-os/cosmic-epoch#975]
So, the compositor can make any Wine app window, and draw the things on it, but i try to interact with the window, nothing happends, the cursor go below the window, and sometimes (if you take the window to the dock and the open it back) this will go black and the drawn thing not seeable.

**Comment by Drakulix** [comment]
It's clearly not every wine app, given many games work fine and I just tested a bunch of EXE files on my system.

Could you perhaps go into some more detail? What applications did you try specifically? Which wine version are you running? Are you running wine through some tool? (E.g. Bottles, PlayOnLinux, Lutris, etc...)

**Comment by fablevi** [comment]
Yes, so, i tried notepad++ installer via lutris, i tried any game in heroic game launcher with Wine-GE-Proton8-26, and a cracked skyrim with proton 9.0-3.

These all working on Gnome/Fedora 40.
